extensions/wikia/Tasks/proxy/proxy.php

```diff
@@ -52,11 +52,19 @@
 $request = new FauxRequest( $_POST, true );
 
 // finally, execute the task
-$runner = Wikia\Tasks\TaskRunner::newFromRequest( $request );
-$runner->run();
+try {
+	$runner = Wikia\Tasks\TaskRunner::newFromRequest( $request );
+	$runner->run();
+	$resp = $runner->format();
+} catch ( Throwable $ex ) {
```

Comment:
$exception?

Reply:
Both, `$ex` and `$exception` are used across the codebase. Personally, I prefer shorter. `Throwable` provides enough information. As a matter of fact `$e` would be even better, as `Throwable` may be either an `Exception` or `Error` :)